client/src/components/views/Challenge/Modal.tsx

```diff
@@ -11,24 +12,35 @@ const Modal = ({ challengeTitle, isOpen, setIsOpen, modalBodyRef }: { challengeT
         setIsOpen(false);
     }, [setIsOpen])
     const [solvers, setSolvers] = useState<GetSolversResponseDtoSolversInner[]>([]);
-    const [pages, setPages] = useState<number>(0);
+    const [page, setPage] = useState<number>(1);
     const { auth } = useAuthContext();
+    const [totalPages, setTotalPages] = useState<number>(0);
     useEffect(() => {
         if (!auth || !challengeTitle) {
             return;
         }
         new LogApi().logControllerGetSolvers({
             accessToken: auth.token,
             challengeTitle: challengeTitle,
-            page: pages! - 1
+            page: page! - 1
         }).then((response) => {
             const { solvers, pages } = response.data;
             setSolvers(solvers);
-            setPages(pages);
-            console.log(solvers, pages);
+            setTotalPages(pages);
+        }).catch((error) => {
+            console.log(error);
         })
     }, [auth]);
```

Comment:
There is no dependence on the page. Does this work?

Reply:
modify